src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -192,9 +192,10 @@ private enum Flags : byte
                     }
 
                     string fieldName = GeneratedNames.MakeBackingFieldName(_sourceName);
+                    bool isInitOnly = setSyntax?.Keyword.IsKind(SyntaxKind.InitKeyword) == true;
```

Comment:
>setSyntax?.Keyword.IsKind(SyntaxKind.InitKeyword) == true; [](start = 38, length = 58)

Consider also checking if the property static.

Reply:
Will leave as-is. We can discuss how we want the API to behave in error cases later. PROTOTYPE tracks IsInitOnly API

---
In reply to: [415127919](https://github.com/dotnet/roslyn/pull/43275#discussion_r415127919) [](ancestors = 415127919)